### PR TITLE
[Impeller] Make //impeller/display_list TU naming consistent.

### PR DESCRIPTION
**Pull request by chinmaygarde**
No change in functionality. I've just renamed the TU's to replace `display_list_` to `dl_`. I am sure there are chances to make the naming better. For instance, the op receiver still being named a dispatcher.

### CODE REVIEW

File: impeller/display_list/dl_dispatcher.h [hunk 1]
```diff
@@ -11,11 +11,11 @@
 
 namespace impeller {
 
-class DisplayListDispatcher final : public flutter::DlOpReceiver {
+class DLDispatcher final : public flutter::DlOpReceiver {
```
Comment (flar) [inline impeller/display_list/dl_dispatcher.h:14]: DisplayList uses Dl as a prefix with a lower case L.

Also, perhaps DlImpellerDispatcher to match DlSkCanvasDispatcher?

**Review by flar — COMMENTED** [review]
My comments are "food for thought" at this point.

**Comment by chinmaygarde** [comment]
> DisplayList uses Dl as a prefix with a lower case L.

Whoops. Fixed and made it consistent.

> Should they have their own naming convention as they are the Impeller implementations of some of the DL interfaces rather than actual APIs/classes provided by DL?

I think being consistent with DL is fine as thats the interface this module is trying to adhere to.

**Review by flar — APPROVED** [review]
LGTM

**Pull request merged**